packages/material-ui/src/styles/createTransitions.d.ts

```diff
@@ -49,3 +49,8 @@ export interface Transitions {
   create: typeof create;
   getAutoHeightDuration: typeof getAutoHeightDuration;
 }
+
+export default function createTransitions(inputTransitions?: {
```

Comment:
The test added does not require this change. Are you sure this is a public method?

Reply:
I know that `import createTransitions from '@material-ui/core/styles/createTransitions` is not what we document but I have seen some project do it and me also in the past. At first I am reluctant to open this PR but then I check other function like `createTypography`, ...`create...`, they all export default from the file.

Reply:
> is not what we document but I have seen some project do it and me also in the past.

We don't support this for a reason. So let's not work on unsupported usages.

Reply:
Agree with @eps1lon on this one. It is not a public API, and it should not be used as such.

Reply:
Okay, then I will close this PR.